plugin/modelgen/models_test.go

```diff
@@ -14,8 +14,14 @@ import (
 func TestModelGeneration(t *testing.T) {
 	cfg, err := config.LoadConfig("testdata/gqlgen.yml")
 	require.NoError(t, err)
+<<<<<<< HEAD
```

Comment:
conflict metadata

Reply:
Ah.  Thanks for finding that.  Will fix.

Reply:
Fix pushed up.  All tests passed.